test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/MinimizedTagHelpers.cshtml

```diff
@@ -15,4 +15,5 @@
                 input-unbound-required="hello2" 
            catchall-unbound-required 
            input-bound-required-string="world" />
+    <div boundbool booldict-prefix-key></div>
```

Comment:
Do we have tests validating TagHelpers with an unmatched bound boolean attribute? Technically there should be no codegen for that but want to verify we're not missing any explosion cases

Reply:
Just added a line below this.